core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -232,7 +232,17 @@ export class ContributorAdminStatsTable implements OnInit {
   getUpperLimitValueForPagination(): number {
     return Math.min(
       this.statsPageNumber * this.itemsPerPage + this.itemsPerPage,
-      this.statsPageNumber * this.itemsPerPage + this.dataSource.length
+      this.allStats.length
+    );
+  }
+
+  showMoreThanMaximumExpectedItems(): string {
```

Comment:
I think it's better to just inline this function. The issue with the current name is that it sounds like calling it will show something, but really it just returns a string.

It's hard to know what to name that string, though, so I suggest just inlining the logic in getOverallItemCount() if nothing else is using it.

Reply:
Done